rest/rest-resources/src/main/resources/OSGI-INF/blueprint/service.xml

```diff
@@ -75,6 +75,7 @@ limitations under the License.
     <bean id="apidocResourceBean" class="org.apache.brooklyn.rest.resources.ApidocResource"/>
     <bean id="applicationResourceBean" class="org.apache.brooklyn.rest.resources.ApplicationResource"/>
     <bean id="catalogResourceBean" class="org.apache.brooklyn.rest.resources.CatalogResource"/>
+    <bean id="bundlesResourceBean" class="org.apache.brooklyn.rest.resources.BundleResource"/>
```

Comment:
also need `typesResourceBean` for `TypeResource`

Reply:
Good catch! Endpoint added @aledsage